Libraries/Interaction/PanResponder.js

```diff
@@ -290,40 +364,50 @@ const PanResponder = {
    *  accordingly. (numberActiveTouches) may not be totally accurate unless you
    *  are the responder.
    */
-  create: function(config) {
+  create(config: PanResponderConfig) {
     const interactionState = {
       handle: (null: ?number),
     };
-    const gestureState = {
+    const gestureState: GestureState = {
       // Useful for debugging
       stateID: Math.random(),
+      moveX: 0,
+      moveY: 0,
+      x0: 0,
+      y0: 0,
+      dx: 0,
+      dy: 0,
+      vx: 0,
+      vy: 0,
+      numberActiveTouches: 0,
+      _accountsForMovesUpTo: 0,
     };
-    PanResponder._initializeGestureState(gestureState);
     const panHandlers = {
-      onStartShouldSetResponder: function(e) {
-        return config.onStartShouldSetPanResponder === undefined
+      onStartShouldSetResponder(event: PressEvent) {
```

Comment:
If you type this as returning a boolean I think you'll find that the flow type for `onStartShouldSetPanResponder` is not correct.

Same with `onStartShouldSetResponderCapture` and `onResponderTerminationRequest`

Reply:
👍 will take a look at this. Also I realize that the type for `touchHistory.touchBank` is wrong, so I will fix that too.